symbols/arm9.yml

```
@@ -2243,30 +2354,102 @@ arm9:
-    - name: RemoveItemsTypeInStorage
+    - name: GetEquivBulkItemIdxInStorage
+      address:
+        NA: 0x200FF50
+      description: |-
+        Checks for a storage item equivalent to the bulk_item and returns the index of the item in storage.
+        Returns -1 if unable to find an equivalent item.
+        
+        r0: pointer to a bulk_item
+        return: index in storage
+    - name: ConvertStorageItemAtIdxToBulkItem
+      address:
+        NA: 0x200FFAC
+      description: |-
+        Get an item in storage and converts it into an equivalent bulk_item. This does not remove the
+        item from storage.
+        
+        r0: item index
+        r1: [output] pointer to a bulk_item
+        return: bool whether or not the item id is not 0
+    - name: ConvertStorageItemAtIdxToItem
+      address:
+        NA: 0x200FFF4
+      description: |-
+        Get an item in storage and converts it into an equivalent item. The item does NOT have the exists
+        flag set to true. This does not remove the item from storage.
+        
+        r0: item index
+        r1: [output] pointer to an item
+        return: bool whether or not the item id is not 0
+    - name: RemoveItemAtIdxInStorage
+      address:
+        NA: 0x20101A0
+      description: |-
+        Remove an item at the specified index from storage.
+        
+        r0: storage item idx
+    - name: RemoveBulkItemInStorage
       address:
         EU: 0x201028C
         NA: 0x20101E4
         JP: 0x20101B4
       description: |-
+        Removes a storage item equivalent to the bulk_item passed from storage.
         Probably? Implements SPECIAL_PROC_0x2A (see ScriptSpecialProcessCall).
         
-        r0: pointer to an owned_item
-        return: bool
-    - name: AddItemToStorage
+        r0: pointer to a bulk_item
+        return: bool whether an item was removed
+    - name: RemoveItemInStorage
+      address:
+        NA: 0x2010260
+      description: |-
+        Removes a storage item equivalent to the item passed from storage.
+        
+        r0: pointer to an item
+        return: bool whether an item was removed
+    - name: StorageZInit
+      address:
+        NA: 0x20102DC
+      description: |-
+        Initializes the storage to be empty.
+        
+        No params.
+    - name: AddBulkItemToStorage
       address:
         EU: 0x20103C4
         NA: 0x201031C
         JP: 0x20102EC
       description: |-
+        Attempts to add the bulk_item to storage.
         Implements SPECIAL_PROC_ADD_ITEM_TO_STORAGE (see ScriptSpecialProcessCall).
         
-        r0: pointer to an owned_item
-        return: bool
+        r0: pointer to a bulk_item
+        return: bool whether an item was removed
+    - name: AddItemToStorage
+      address:
+        NA: 0x20103AC
+      description: |-
+        Attempts to add the item to storage.
+        
+        r0: pointer to an item
+        return: bool whether an item was removed
```

Should this be:

```suggestion
        return: bool whether an item was added
```
?